SPEC.md

```diff
@@ -106,6 +106,11 @@ This <ins>optional</ins> key’s value (type string) shall contain the PCI addre
 of the associated physical interface (in standard BDF format “dddd:BB:DD.f”) if
 available and applicable.
 
+#### 3.1.3.5 “vf-representor-device”
```

Comment:
shall we go with: "device-representor" or "representor-device" ?
im thinking that down the road we would want to reuse this field name for subfunction[1] devices as well as VF.
(perhaps in a different section/type not sure yet, however id prefer it to have the same name)

WDYT ?

work to support this in sriov network device plugin is already in motion [2]

[1] https://www.kernel.org/doc/html/latest/networking/devlink/devlink-port.html#subfunction
[2] https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/issues/430

Reply:
Sounds good. I update PR to use `representor-device` field name. We already have `rdma-device` field, so I decided to use `representor-device` instead of `device-representor`.

Also, I pushed commit which updates spec version